src/infrastructure/figma/transformers/figma-file-meta-transformer.test.ts

```diff
@@ -42,18 +42,9 @@ describe('transformFileMetaToAtlassianDesign', () => {
 		expect(result).toStrictEqual({
 			id: fileKey,
 			displayName: fileMetaResponse.file.name,
-			url: buildDesignUrl({
-				fileKey,
-				fileName: fileMetaResponse.file.name,
```

Comment:
**note:** Just removes `fileName` as it is unused by the `buildDesignUrl` function (likely, the code was merged incorrectly).